app/template/rest.py

```diff
@@ -211,9 +214,21 @@ def preview_letter_template_by_notification_id(service_id, notification_id, file
         content = base64.b64encode(pdf_file).decode('utf-8')
 
         if file_type == 'png':
-            url = '{}/precompiled-preview.png{}'.format(
-                current_app.config['TEMPLATE_PREVIEW_API_HOST'],
-                '?page={}'.format(page) if page else ''
+
+            try:
+                page_number = page if page else "0"
```

Comment:
I wonder if casting `page` var on line 196 to an int would be better than the casting logic here as it should still work with the format logic for the created letter path